android-components/components/feature/accounts/src/test/java/mozilla/components/feature/accounts/FirefoxAccountsAuthFeatureTest.kt

```diff
@@ -71,6 +73,10 @@ class TestableFxaAccountManager(
 
 @RunWith(AndroidJUnit4::class)
 class FirefoxAccountsAuthFeatureTest {
+    val entryPoint: FxAEntryPoint = mock<FxAEntryPoint>().apply {
```

Comment:
Should we rename `entryPoint` to `mockEntryPoint` as `FxAEntryPoint` is a mocked class. What do you think?